cli/internal/context/testdata/duplicate-pkgs/package.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "repo-with-dupe-workspaces",
```

I think you changed this without regenerating the lockfile:
```suggestion
  "name": "duplicate-pkgs",
```